docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -51,7 +58,9 @@ That's all you need to start returning log data from your data source. Go ahead
 
 Congratulations, you just wrote your first logs data source plugin! Next, let's look at a couple of features that can further improve the experience for the user.
 
-### (Optional) Add preferred visualisation type hint to the data frame
+## Enhance your logs data source plugin with optional features
```

Comment:
The writer's toolkit suggests using ["Optional: ..."](https://github.com/grafana/writers-toolkit/blob/main/docs/static/templates/tutorial-template.md) in the heading. Would that be a useful change here? Also, there is only one subheading under the enhancements, so is there only one possible enhancements? If so, do we even need this heading? 

Also, consider adding a sentence or two that summarizes what these optional features are.

Reply:
(Optional) is not in my draft text